esd-group-3/src/java/models/User.java

```diff
@@ -132,4 +132,39 @@ public void retrieveByUserId(DBConnection dbcon, int id) {
             System.out.println(e);
         }
     }
+    
+    public void editUser(DBConnection dbcon, String setDatabase, String username, String toChange, String updatedValue) {            
+        String query = "";
+        
+        if (setDatabase == "Clients" || setDatabase == "Employees") {
+            query = "SELECT id FROM Users WHERE username = '" + username + "'";
+
+            int userid = 0;
+            try (Statement stmt = dbcon.conn.createStatement()) {
+                ResultSet resultSet = stmt.executeQuery(query);
+                while (resultSet.next()) {
+                    userid = resultSet.getInt("id");
+                }
+            } catch (SQLException e) {
+                System.out.println(e);
+            }
+
+            query = "UPDATE Clients SET " + toChange + "='"+ updatedValue +"' WHERE userid =" + userid;
+
+            try (Statement stmt = dbcon.conn.createStatement()) {
+                int resultSet = stmt.executeUpdate(query);
+            } catch (SQLException e) {
+                System.out.println(e);
+            }
```

Comment:
I'm not sure if i'm reading this wrong, but if you choose to update the employees table, won't this change the client table?